lib/hooks/useContributorPullRequestsChart.ts

```diff
@@ -5,7 +5,13 @@ import { getAvatarByUsername } from "lib/utils/github";
 import getFormattedTooltipValue from "lib/utils/get-formatted-tooltip-value";
 import useContributorPullRequests from "./api/useContributorPullRequests";
 
-const useContributorPullRequestsChart = (contributor: string, topic: string, repoIds: number[] = [], range = 30) => {
+const useContributorPullRequestsChart = (
```

Comment:
Refeactor (non-blocking): There's quite a few parameters now. Consider promoting them to one option parameter with all the current parameters.